app/Providers/RepositoryServiceProvider.php

```diff
@@ -7,20 +7,23 @@
 use HMS\Entities\User;
```

Comment:
It might be worth splitting provider into smaller providers. As I can see it getting potentially quite large. 

Maybe a ACL related repo provider
User related repo provider